modules/react-native-status/android/src/main/java/im/status/ethereum/pushnotifications/PushNotificationHelper.java

```diff
@@ -870,7 +870,11 @@ private void cleanGroup(String groupId) {
     public void start() {
         Log.e(LOG_TAG, "Starting Foreground Service");
         Intent serviceIntent = new Intent(context, ForegroundService.class);
-        context.startService(serviceIntent);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            context.startForegroundService(serviceIntent);
+        } else {
+            context.startService (serviceIntent);
```

Comment:
```suggestion
            context.startService(serviceIntent);
```

Reply:
I'm surprised how this was not a compile time error :D

Reply:
wow okay it does seem like this is allowed https://courses.cs.washington.edu/courses/cse142/21su/quality/java-conventions/#spacing

Reply:
wow it does seems like this is indeed valid syntax.. but yeah it would be better if we remove this space to avoid any confusion